ballerina/main.bal

```diff
@@ -0,0 +1,5 @@
+import ballerina/io;
```

Comment:
Let's remove this.

Reply:
is it okay to initialise with an empty bal file?

Reply:
Yes.